include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -28,17 +28,23 @@ def VerbatimExprOp : SVOp<"verbatim.expr", [NoSideEffect, HasCustomSSAName]> {
     sv.verbatim.expr allows operand substitutions with {{0}} syntax.
     }];
 
-  let arguments = (ins StrAttr:$string, Variadic<AnyType>:$operands);
+  let arguments = (ins StrAttr:$string, Variadic<AnyType>:$operands,
+                  DefaultValuedAttr<SymRefArrayAttr,"{}">:$symbols);
   let results = (outs AnySignlessIntegerOrInOutType:$result);
   let assemblyFormat = [{
-    $string attr-dict (`(` $operands^ `)`)?
-      `:` functional-type($operands, $result)
+    $string (`(` $operands^ `)`)?
+      `:` functional-type($operands, $result) attr-dict 
   }];
 
   let builders = [
     OpBuilder<(ins "Type":$resultType, "Twine":$string),
                "build(odsBuilder, odsState, resultType, "
-               "odsBuilder.getStringAttr(string), ValueRange{});">
+               "odsBuilder.getStringAttr(string), ValueRange{}," 
+               "odsBuilder.getArrayAttr({}));">,
+    OpBuilder<(ins "Type":$resultType, "StringAttr":$string, "ValueRange":$operands),
```

Comment:
These builders are changing two different things: presence of operands, and StringAttr vs Twine.  I'd suggest making the builders consistent and only differ on Twine vs StringAttr.

Concretely: add a operand list to the first one (defaulting to `{}` with CArg) and make the second one also default to {}